R/odm-dictionary-file.R

```diff
@@ -0,0 +1,9 @@
+odm_dictionary <- list()
+
+odm_dictionary$dictionary_directory <- "data/raw"
```

Comment:
Aren't the first two variables (dictionary_directory and tmp_dictionary_directory) not applicable to an ODM dictionary but only to the automation? If so, it should be moved to the files-validation.R file.